backend_new/prisma/schema.prisma

```diff
@@ -0,0 +1,1025 @@
+
+  @@map("units_summary")
+}
+
+// Note: [mfa_code] formerly max length 16; [first_name, middle_name, last_name] formerly max length 64
+// Note: [email] formerly lower case enforced
+model UserAccounts {
+  id                       String                  @id() @default(dbgenerated("uuid_generate_v4()")) @db.Uuid
+  passwordHash             String                  @map("password_hash") @db.VarChar
+  passwordUpdatedAt        DateTime                @default(now()) @map("password_updated_at") @db.Timestamp(6)
+  passwordValidForDays     Int                     @default(180) @map("password_valid_for_days")
+  resetToken               String?                 @map("reset_token") @db.VarChar
+  confirmationToken        String?                 @map("confirmation_token") @db.VarChar
+  confirmedAt              DateTime?               @map("confirmed_at") @db.Timestamptz(6)
+  email                    String                  @unique() @db.VarChar
+  firstName                String                  @map("first_name") @db.VarChar
+  middleName               String?                 @map("middle_name") @db.VarChar
+  lastName                 String                  @map("last_name") @db.VarChar
+  dob                      DateTime?               @db.Timestamp(6)
+  phoneNumber              String?                 @map("phone_number") @db.VarChar
+  createdAt                DateTime                @default(now()) @map("created_at") @db.Timestamp(6)
+  updatedAt                DateTime                @updatedAt @map("updated_at") @db.Timestamp(6)
+  language                 LanguagesEnum
+  mfaEnabled               Boolean                 @default(false) @map("mfa_enabled")
+  mfaCode                  String?                 @map("mfa_code") @db.VarChar
+  mfaCodeUpdatedAt         DateTime?               @map("mfa_code_updated_at") @db.Timestamptz(6)
+  lastLoginAt              DateTime                @default(now()) @map("last_login_at") @db.Timestamp(6)
+  failedLoginAttemptsCount Int                     @default(0) @map("failed_login_attempts_count")
+  phoneNumberVerified      Boolean?                @default(false) @map("phone_number_verified")
+  agreedToTermsOfService   Boolean                 @default(false) @map("agreed_to_terms_of_service")
+  hitConfirmation_url      DateTime?               @map("hit_confirmation_url") @db.Timestamptz(6)
+  activeAccessToken        String?                 @map("active_access_token") @db.VarChar
+  activeRefreshToken       String?                 @map("active_refresh_token") @db.VarChar
+  activityLogs             ActivityLog[]
+  applicationFlaggedSet    ApplicationFlaggedSet[]
+  applications             Applications[]
+  listings                 listings[]
+  jurisdictions            Jurisdictions[]
+  userPreferences          UserPreferences?
+  userRoles                UserRoles?
+
+  @@map("user_accounts")
+}
+
+model UserRoles {
```

Comment:
We had two classes in this file, not sure if we "lost" the one that isn't an entity or need it

Reply:
we'll need it but its only important for DTO purposes not db purposes

Reply:
or actually we may not need it, I haven't quite thought out how its being used but tbh we may be able to just roll that into something else/prisma may have a better answer for us